pdg/src/util.rs

```diff
@@ -113,15 +113,6 @@ impl<T: Display> Display for Duplicates<T> {
     }
 }
 
-impl<T: Debug + Eq + Hash> Duplicates<T> {
-    pub fn assert_empty_debug(&self) {
-        if self.is_empty() {
-            return;
-        }
-        panic!("unexpected duplicates: {:?}", self);
-    }
-}
-
```

Comment:
Don't delete this.  It's sometimes used, and it has nothing to do with this PR.

Reply:
It issues a dead code warning. If it is occasionally used, I'll add an allow with a comment.

Reply:
We haven't fixed all the warnings yet, so can you just leave this as is?  This is in fast-changing PDG code.